client/scss/components/_header.scss

```diff
@@ -36,6 +37,11 @@
     font-weight: 400;
   }
 
+  .w-header__description {
+    font-size: 0.6em;
+    font-weight: 400;
+  }
```

Comment:
Can we try to use existing theme variables here?

Reply:
I don't remember any variables for font size and weight. Most of the places I've looked at are using hardcoded values